scripts/coverage/config.json

```diff
@@ -1,30 +1,6 @@
 {
   "ignorePatterns": [
-    "AppDelegate",
-    "Cell.\\w+$",
-    "Controller.\\w+$",
-    "Transitioning.\\w+$",
-    "View.\\w+$",
-    "Window.\\w+$",
-    "TabBar.\\w+$",
-    "\\/Views\\/",
-    "CanvasCore\/",
-    "CanvasKit1\/",
-    "Core\/ShareExtension\/",
-    "Core\/Use Cases\/SubmitFileSubmission",
-    "Frameworks\/",
-    "OperationQueueExtensions.swift",
-    "Parent\/",
-    "Pods\\/",
-    "Student\/Canvas\/",
-    "Student\/Courses\/CourseList\/",
-    "Student\/Dashboard\/",
-    "Student\/GradesWidget\/",
-    "TechDebt\/",
-    "UITestHelpers\/",
-    "rn/Teacher\/src\/modules\/graphql-speed-grader\/",
-    "SubmissionButtonPresenter.swift"
```

Comment:
I think we'll still want some exclusions for danger reporting. I don't think we intend to add tests for any of the following:

```
    "AppDelegate",
     "CanvasCore\/",
     "CanvasKit1\/",
     "Frameworks\/",
     "Parent\/",
     "Pods\\/",
     "Student\/Canvas\/",
     "TechDebt\/",
     "UITestHelpers\/"
```

well, Parent eventually, but not yet.